### DISCUSSION TITLE
Is cockroachdb formally supported?

### DISCUSSION THREAD

**Discussion by xu4wang** [directus/directus#17089] [Archive]
I am choosing a DB for a new Directus project, to make the solution scalable, I am thinking to use cockroachdb.

Is it formally supported by directus? is anybody using it in production already? 

BR,Austin

**Comment by paescuj** [comment]
> Is it formally supported by directus?

Yes, it is: https://docs.directus.io/self-hosted/docker-guide.html#supported-databases

> is anybody using it in production already?

Reading other discussions here and on Discord, I have the strong impression that this is the case, but I can't tell you for sure.
